### PR TITLE
Head line: add cli implement first version.

### PR DESCRIPTION
**Pull request by Yihen-Liu**
add cmd package for cli module.
  restruct main.go file.

  Signed-off-by:liuyihen@onchain.com

### CODE REVIEW

File: account/client.go [hunk 1]
```diff
@@ -426,11 +434,18 @@ func GetClient() Client {
...
-	if c == nil {
+
+	var client Client
+	if "" != wallet {
+		client = Open(wallet, passwd)
+	} else {
+		client = Open(WALLET_FILENAME, passwd)
+	}
+
+	if client == nil {
```
Comment (laizy) [inline account/client.go]: meaningless condition check , return client directly.

File: cmd/info_cmd.go [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+	ledgerPID := ldgerActor.Start()
+	actor.SetLedgerPid(ledgerPID)
+	height, _ := actor.CurrentBlockHeight()
+	fmt.Println("Current blockchain height: ", height)
+	return nil
+}
+
+func blockInfoCommand(ctx *cli.Context) error {
+	/*init ledger actor for info command*/
+	ledger.DefLedger, _ = ledger.NewLedger()
```
Comment (laizy) [inline cmd/info_cmd.go:118]: info should query from restful api , can not get from db directly

**Review by laizy — CHANGES_REQUESTED** [review]

File: cmd/info_cmd.go (hunk 2 above)
Comment (laizy) [reply cmd/info_cmd.go:118]: because command is executed in a separate process

Comment (Yihen-Liu) [reply cmd/info_cmd.go:118]: Thx.

File: cmd/utils/flags.go [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+	}
+
+	TestnetFlag = cli.BoolFlag{
+		Name:  "testnet",
+		Usage: "Ropsten network: pre-configured proof-of-work test network",
+	}
+
+	RinkebyFlag = cli.BoolFlag{
+		Name:  "rinkeby",
+		Usage: "Rinkeby network: pre-configured proof-of-authority test network",
```
Comment (laizy) [inline cmd/utils/flags.go:67]: what's this?

**Review by laizy — CHANGES_REQUESTED** [review]

**Comment by Yihen-Liu** [comment]
New version has been given pull request.
Close this one.

**Pull request closed without merging**